app/views/maintenance.scala.html

```diff
@@ -0,0 +1,19 @@
+@import models.user.User
+@(title: String, user: Option[User] = None)(implicit lang: Lang)
```

Comment:
We don't need to include the `user` object here. You can remove the parameter and the import.